contracts/MirrorNode.sol

```diff
@@ -40,6 +40,14 @@ abstract contract MirrorNode {
         return "";
     }
 
+    function getNftCreatedTimestamp(address token, uint32 serial) external returns (string memory) {
+        string memory json = this.fetchNonFungibleToken(token, serial);
```

Comment:
let's rename `fetchNonFungibleToken` to `fetchNonFungibleTokenSerial`. This is to avoid confusion between the NFT and the actual token (which includes the serial). Also to be consistent with the rest.